src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenShortCircuitOperatorTests.cs

```diff
@@ -5865,7 +5865,7 @@ .maxstack  2
         }
 
         [WorkItem(825, "https://github.com/dotnet/roslyn/issues/825")]
-        [Fact]
+        [ConditionalFact(typeof(IsEnglishLocal))]
```

Comment:
This test should probably be rewritten to not need the conditional fact.  Is there any way to get the localization info from the exception?

https://github.com/dotnet/roslyn/issues/5662